students/hayoung-gam/controllers/usersController.js

```diff
@@ -0,0 +1,37 @@
+import { PrismaClient } from '@prisma/client';
+
+const prisma = new PrismaClient();
+
+export const getUsersList = async (req, res) => {
+  try {
+    const users = await prisma.$queryRaw(`SELECT * FROM users;`);
+
+    res.status(201).json({ users });
+  } catch (err) {
+    res.status(500).json({ message: err.message });
+  }
+};
+
+export const signUp = async (req, res) => {
+  try {
+    const { email, name, password } = req.body;
+
+    const user = await prisma.$queryRaw(`
+      SELECT * FROM users WHERE email='${email}';
+    `);
+
+    if (user.length) {
+      res
+        .status(400)
+        .json({ message: 'THIS_EMAIL_ADDRESS_IS_ALREADY_BEING_USED' });
+    }
+
+    const createdUser = await prisma.$queryRaw(
+      `INSERT INTO users(email, name, password) VALUES ('${email}', '${name}', '${password}');`
+    );
+
```

Comment:
```suggestion

    const createdUser = await prisma.$queryRaw(
      `INSERT INTO users(email, name, password)
      VALUES ('${email}', '${name}', '${password}');`
    );

```
SQL 문이 길어질 때에는 SQL 문법이 시작되는 곳에서 줄을 분리해주시면 가독성이 좋습니다 :)

Reply:
와우 조언 감사합니다! 코드를 수정했습니다!

https://github.com/wecode-bootcamp-korea/fullstack-1-westarbucks-backend-team1/pull/5/commits/a4a606a41e761dca48ab38268bc097ab96a3044e